source/macros.tex

```diff
@@ -634,4 +641,6 @@
 \let\addcontentsline\oldcontentsline%
 }
 \newcommand{\defncontext}[1]{\textlangle#1\textrangle}
-\newenvironment{defnote}{\addtocounter{termnote}{1}\noteintro{Note \thetermnote{} to entry}}{\noteoutro{note}\space}
+\newenvironment{defnote}
```

Comment:
While we're at it, shouldn't this get a \par in front, too?

Reply:
Done, and duplication between this and note/example eliminated too.